app/views/index/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div id="app">
+</div>
+
+<h1>hello alex<h1>
```

Comment:
this confuses me. There should not be an `index` folder.  Im not even sure what would render this? Additionally, you would want to remove the `hello alex`

Instead, this should maybe be in a static_pages folder, and you should have a specifc action in your controller refer to it. Lots of different ways to implement it, but right now Im not sure what this does